tests/integrations/subprocess/test_Subprocess__stream_output.py

```diff
@@ -13,7 +21,7 @@ def popen_process():
     # continuously returning "" once it reaches EOF
     process.stdout.readline.side_effect = [
         "output line 1\n",
-        "\n",
+        sleep(0.25) or "\n",  # sneaky method to slow the output streamer down
```

Comment:
Are you sure this works the way you think it does? The sleep will be evaluated when the side effect is defined, not when the side effect is realised; this would need to be in a lambda to be deferred. The only impact of this line will be to make the setup of the test case slower.

Reply:
Ahhh....of course....i must have been working on this too long when I thought this would help. At any rate, I think there may still be race condition for whether codecov thinks the inner `sleep` in `stream_output` is covered or not....not a big deal either way.